internal/image_from_checkpoint_archive.go

```diff
@@ -0,0 +1,142 @@
+package internal
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	metadata "github.com/checkpoint-restore/checkpointctl/lib"
+)
+
+const (
+	BUILD_SCRIPT  = "/usr/libexec/build_image.sh"
+	PODMAN_ENGINE = "libpod"
```

Comment:
I am curious about where we will need these exported Constants. Maybe PODMAN_ENGIME could be used [here](https://github.com/checkpoint-restore/checkpointctl/blob/15dd6b5ae2e32f3435650f15826dd367a70fc984/internal/container.go#L184). But what about BUILD_SCRIPT?